ui/partials/main/tcjobactions.html

```diff
@@ -12,7 +12,8 @@ <h4 class="modal-title">Custom Taskcluster Job Actions</h4>
     </select>
     <p id="selectedActionHelp" class="help-block">{{input.selectedAction.description}}</p>
   </div>
-  <div class="form-group">
+  {{input.selectedAction.schema}}
```

Comment:
Hah, sorry, that's debugging output.  Funny that I left it in a review for almost three weeks..